builds/jazz-build-module/scm-module.groovy

```diff
@@ -338,4 +338,27 @@ def getRepoCommitterInfo(commitHash) {
     }
 }
 
+def updateAclPermission(serviceId, policiesList, auth_token, aclUrl) {
+	try {
+		def body = JsonOutput.toJson([
+			serviceId: serviceId,
+			policies: policiesList
+		]);
+		def updatePermission = sh(script: "curl -X POST \
+				${aclUrl} \
+				-k -v -H \"Authorization: $auth_token\" \
+				-H \"Content-Type: application/json\" \
+				-d \'${body}\'", returnStdout: true).trim()
+		def responseJSON = parseJson(updatePermission)
+
+		if (responseJSON && responseJSON.success) {
+			echo "Successfully updated permissions for code and deploy."
```

Comment:
I think you are  updating permissions specific  to code and deploy. You can remove the part - `code and deploy` here.

Same with error below.

Reply:
Updated the message.